### DISCUSSION TITLE
environment variable list

### DISCUSSION THREAD

**Discussion by flatmax** [Igalia/cog#526] [Q&A]
Are the run time environment variables listed anywhere ?

**Comment by Geroge-W** [comment]
https://trac.webkit.org/wiki/EnvironmentVariables

**Comment by flatmax** [comment]
Thanks for the information. I am pretty interested in running cog without any desktop environment -> no weston/wayland and no Xorg.

Those environment vairables don't list methods for altering the DRM subsystem. Are there any ways to control that platform ?

**Reply by aperezdc** [reply]
Additional parameters can be passed to the DRM/KMS plug-in (programmatically, or using a configuration file). Those are listed in the plug-in documentation: https://igalia.github.io/cog/platform-drm.html

There are also a few environment variables which were not written down yet, so I have created https://github.com/Igalia/cog/pull/530 to add them to the documentation.

**Comment by philn** [comment]
`cog -P drm http://example.com`
